sw/device/lib/testing/random.c

```diff
@@ -0,0 +1,55 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/testing/random.h"
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "sw/device/lib/testing/check.h"
+
+/**
+ * The initial LFSR seed value.
+ *
+ * The initial LFSR seed used for generating pseudo-random numbers is set to 1.
+ * This symbol can be overridden in DV to match the same seed as the one used
+ * passed to the ASIC simulator. This allows the DV tests to be run with
```

Comment:
How? How do you know the compiler won't inline it? Why can't this be done through device.h?

Reply:
Moved to device.h.

Reply:
Looks like C does not allow you to initialize a static variable with a const variable - its needs to be a const literal. I will find some other way to modify initial `lfsr` value later.

Reply:
Can you file a TODO bug?